### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Name, Role and Value] [MAS4.1.2][HighImpact][ Role is not defined for all the Interactive controls in Iframe.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#698]
**User Experience:**
User who rely on AT will not get proper information about the elements, if  role is not defined properly which provides the purpose of control.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Developer tool

**Attachments:**
![mas4 1 2_role is not defined_iframe](https://user-images.githubusercontent.com/33054846/32538312-93cd5b00-c48b-11e7-85e8-85067bb25dbb.png)
![mas4 1 2 role_iframe](https://user-images.githubusercontent.com/33054846/32541583-11a87618-c496-11e7-8eb8-bc2441123558.png)

**Repro Steps:**
1.	Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2.	Navigate to “report generation” Iframe  and open developer tool.
3.	Verify Role is defined for all the Interactive controls in Iframe  or not.

**Current Behavior:**
Role is not defined for all the controls in Iframe.

**Expected Behavior:**
Role should be defined for all the controls in Iframe which helps the user to get to know the purpose of control. Role should be defined as Tab for header controls (Script, Template, Style, Libraries) and Button for Dark and open in script lab control.

**Note:**
1. State(selected/Not selected) also should be defined for header controls (Script, Template, Style, Libraries) as it will get selected after pressing enter.
2. Same Issue is also repro for the MAC +Safari [BuildInfo-OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1]. Role is not defined for the all the interactive controls in the iframe and state is not defined for the header controls (Script, Template, Style, Libraries)
3. Name is not defined for icons "Run"," Delete","sign In Github","Share".Thus screen reader do not read anything when focus lands on the icons in the scan/item mode.
4. All the above issues are repro for the Entry Point Page.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.1.2
